array_analyzer/load/report.py

```diff
@@ -21,6 +22,10 @@ def write_od_to_plate(data, well_name, array_type):
     else:
         raise AttributeError(f"well name {well_name} is not recognized")
 
+    if data is None:
+        print("\tDATA IS NONE FOR THIS WELL")
```

Comment:
Should this be a warning? Also, if there's no data, what happens with this array of ones?

Reply:
interesting.  I think this is a remnant from a test I was doing to see if None types was actually the problem.  Basically this condition is never met because this `write_od_to_plate` is called only if data is not None.
Will remove and test